CHANGELOG.md

```diff
@@ -8,6 +8,7 @@
 
 ### Added
 
+- Update `civis_ml` documentation with CivisML 2.1 features and defaults.
```

Comment:
I think we should briefly summarize what they are so it's easy to see at a glance.